code/modules/reagents/machinery/chem_master.dm

```diff
@@ -191,7 +191,8 @@
 					return
 				var/bottle_label = reject_bad_text(input(user, "Label:", "Enter desired bottle label", null) as text|null)
 				var/obj/item/storage/pill_bottle/I = new/obj/item/storage/pill_bottle
-				I.icon_state = "pill_canister"+pillbottlesprite
+				if(pillbottlesprite == "2")//if the 2nd sprite is selected, use 2nd sprite
+					I.set_greyscale_config(/datum/greyscale_config/pillbottleround)
```

Comment:
Im confused, what happens if this is not two?

Reply:
1 is normal pill bottle, 2 is round pill bottle